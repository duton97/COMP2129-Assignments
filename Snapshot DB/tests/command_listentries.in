LIST ENTRIES
SET a 1 2
SET b 2 3
LIST ENTRIES
BYE
