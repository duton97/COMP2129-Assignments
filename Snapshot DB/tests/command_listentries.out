> no entries

> ok

> ok

> b [2 3]
a [1 2]

> bye
